src/modules/InviteUser/components/invitationActions.vue

```diff
@@ -0,0 +1,100 @@
+<script>
+import { reInviteUser, revokeUser } from "../services";
+export default {
+  props: {
+    name: {
+      type: String,
+      required: true,
+    },
+    email: {
+      type: String,
+      required: true,
+    },
+  },
+  methods: {
+    async reInviteMember() {
+      try {
+        const response = await reInviteUser({
+          name: this.name,
+          email: this.email,
+        });
+        //display message coming from response
+        console.log("reinvitation mail sent successfully");
+        alert("Reinvitation mail sent successfully");
+      } catch (error) {
+        alert("error msg:", error.message);
+        console.log("error:", error.message);
```

Comment:
remove "console.log"

Reply:
will remove along with alert message and will be replaced by a conformation box